packages/client/src/App/AppComponent.tsx

```diff
@@ -87,14 +89,22 @@ export const AppComponent: React.FC<Props> = memo(({ darkTheme, toggleDarkTheme
   return (
     <Grid className={`App ${classes.app}`} direction="column" container>
       <AppBar position="relative" classes={{ root: classes.navbar }}>
-        <Toolbar>
+        <Toolbar className={classes.navInnerContainer}>
           <Grid container alignItems="center">
             <img src={wmcIcon} alt="World Memory Challenge Icon" className={classes.wmcIcon} />
-            <Typography variant="h6" color="inherit" className={classes.title} noWrap>
-              <Link href="/" color="inherit" className={classes.titleStyle}>
-                World Memory Challenge
-              </Link>
-            </Typography>
+            {isWidthDown('md', width) ? (
+              <Typography variant="h6" color="inherit" className={classes.title} noWrap>
+                <Link href="/" color="inherit" className={classes.titleStyle}>
+                  WMC
+                </Link>{' '}
```

Comment:
I guess this extra space `{' '}` can be removed?

Reply:
Yes, definitely. Don't really know how it got there in the first place.